### PR TITLE
created filereader.cpp and filereader.h / Parser

### PR DESCRIPTION
**Pull request by fahimbhuiyan**
class that reads an input file containing data about scheduled airplane flights, and extracts the relevant information into an internal schedule vector of aircraft objects.

### CODE REVIEW

**Comment by fahimbhuiyan** [comment]
issue #2

File: .DS_Store
Comment (Vasqq) [inline .DS_Store:1]: Don't need to commit this.

File: .vscode/c_cpp_properties.json
Comment (Vasqq) [inline .vscode/c_cpp_properties.json:1]: Don't need to commit this.

File: Air_Traffic_Control_System/.DS_Store
Comment (Vasqq) [inline Air_Traffic_Control_System/.DS_Store:1]: Don't need to commit this.

File: Air_Traffic_Control_System/src/.DS_Store
Comment (Vasqq) [inline Air_Traffic_Control_System/src/.DS_Store:1]: Don't need to commit this.

File: Air_Traffic_Control_System/src/FileReader.cpp [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+	for (Aircraft plane : schedule) {
+		//LOG_DEBUG("Planes verifyng %d: %d\n", plane.getId(), plane.timestamp);
+
+		if (plane.timestamp == timeVal) {
+			ret.push_back(plane);
+		}
+	}
+
+	//LOG_DEBUG("Planes in cycle %d: %d\n",timeVal, ret.size());
```
Comment (Vasqq) [inline Air_Traffic_Control_System/src/FileReader.cpp:84]: Remove dead code.

File: Air_Traffic_Control_System/src/FileReader.h [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+a long integer representing a time value as an argument, iterates over the schedule vector, 
+and returns a vectorof Aircraft objects representing the scheduled flights that are 
+expected to enter at the specified time.
+*/
+	// iterate over the schedule file, returns aircraft entering at time time
+	std::vector<Aircraft> getScheduledAircraftsForTime(long time);
+
+};
+
+#endif /* SRC_FILEREADER_H_ */
```
Comment (Vasqq) [inline Air_Traffic_Control_System/src/FileReader.h:37]: here

File: Air_Traffic_Control_System/src/FileReader.h [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ * FileReader.h
+ *
+ *  Reads input file
+ *
+ *  Created on: March 15, 2023
+ *      Author: Fahim
+ */
+
+#ifndef SRC_FILEREADER_H_
+#define SRC_FILEREADER_H_
```
Comment (Vasqq) [inline Air_Traffic_Control_System/src/FileReader.h:11]: here

File: Air_Traffic_Control_System/src/FileReader.h [hunk 4]
```diff
@@ -0,0 +1,37 @@
+/*
+ * FileReader.h
+ *
+ *  Reads input file
+ *
+ *  Created on: March 15, 2023
+ *      Author: Fahim
+ */
+
+#ifndef SRC_FILEREADER_H_
```
Comment (Vasqq) [inline Air_Traffic_Control_System/src/FileReader.h:10]: here

File: Air_Traffic_Control_System/src/FileReader.h
Comment (Vasqq) [inline Air_Traffic_Control_System/src/FileReader.h:1]: Change name to Parser.h
** DON'T FORGET TO CHANGE THE #ifndef AND #define AS WELL!

ex// SRC_FILEREADER_H_ becomes SRC_PARSER_H_

File: Documentation/.DS_Store
Comment (Vasqq) [inline Documentation/.DS_Store:1]: Don't need to comit this.

**Review by Vasqq — CHANGES_REQUESTED** [review]
Please see comments and delete all the .dstore files along with the .json file and re commit so we don't merge them to main.

File: Air_Traffic_Control_System/src/FileReader.cpp (hunk 1 above)
Comment (fahimbhuiyan) [reply Air_Traffic_Control_System/src/FileReader.cpp:84]: done

File: Air_Traffic_Control_System/src/FileReader.h (hunk 4 above)
Comment (fahimbhuiyan) [reply Air_Traffic_Control_System/src/FileReader.h:10]: done

File: Air_Traffic_Control_System/src/FileReader.h (hunk 3 above)
Comment (fahimbhuiyan) [reply Air_Traffic_Control_System/src/FileReader.h:11]: done

**Pull request closed without merging**